src/RAWcookedBundle/Resources/views/Default/index.html.twig

```diff
@@ -0,0 +1,292 @@
+                Sponsored by <a href="https://reto.ch">AV Preserve by reto.ch</a> with support from <a href="http://www.cna.public.lu">CNA (National Audiovisual Centre of Luxembourg)</a> and <a href="https://www.nb.no">Nasjonalbiblioteket (National Library of Norway)</a>; designed, led and fully developed by MediaArea.
+            </div>
+
+
+        </div>
+    </div>
+</section>
+
+<section>
+<h2>A current reality of many archives</h2>
+<p>Many archives preserve their single-image based files as TIFF or DPX, grouped by entire films or single reels, and stored inside folders, or TAR or ZIP archives. Yet to have a single file format which is suitable for the preservation of both film and video has been a dream of many audio-visual archives, especially in smaller ones, since the beginning of digital moving images.</p>
+<p>Additional context:</p>
+<ul>
+<li>Reto Kromer: «<a href="https://retokromer.ch/publications/JFP_96.html" target="_blank">Matroska and FFV1: One File Format for Film and Video Archiving?</a>», in <i>Journal of Film Preservation</i>, n. 96 (April 2017)</li>
+<li>Kieran O’Leary: «<a href="https://kieranjol.wordpress.com/2016/10/07/introduction-to-ffv1-and-matroska-for-film-scans/" target="_blank">Introduction to FFV1 and Matroska for Film Scans</a>», 2016–10–07</li>
+</ul>
+<h2>The missing piece of software</h2>
+<p>The solution we propose is <i>RAWcooked</i>, a piece of software that  encodes from and decodes to various so-called «raw» audio-visual file formats in a completely transparent way. The image content and/or sound content, including all the related metadata and sidecar files, is fully preserved bit-by-bit.<br />
+For instance, the unplayable ZIP files could be replaced by a playable file, with the full reversibility of the original package content.</p>
+<h2>Decoding the Matroska file back to the original RAW</h2>
+<p>Whenever needed, <i>RAWcooked</i> decodes back the encoded Matroska file to the original RAW files, including all the original metadata and sidecar files. It is important to stress out clearly that the encoded files can be decoded, and the resulting files are bit-by-bit identical to the original ones. Not only the image and/or sound content is fully preserved, but also all enclosed metadata and the all file’s characteristics. Therefore, an encoded and decoded RAW file cannot be differentiated from its original.</p>
+<h2>A complete ecosystem</h2>
+<p>The Matroska files which have been encoded by <i>RAWcooked</i> can be used on wide range of popular software supporting Matroska/FFV1 and FLAC plackack, including:</p>
+<ul>
+<li><a href="https://www.videolan.org/vlc/">VLC</a> media player for playback</li>
+<li><a href="https://www.ffmpeg.org/">FFmpeg</a> tools can transcode, playback and probe those files</li>
+<li><a href="/QCTools">QCTools</a> can performs in-depth analysis</li>
+<li><a href="/MediaInfo">MediaInfo</a> can display technical information</li>

```

Comment:
Use symfony route : {{ path('mi_home') }} instead of hardcoded path
You can also add MediaConch project